core/src/avm2/globals/json.rs

```diff
@@ -264,7 +264,13 @@ pub fn parse<'gc>(
     _this: Value<'gc>,
     args: &[Value<'gc>],
 ) -> Result<Value<'gc>, Error<'gc>> {
-    let input = args.get_string(activation, 0)?;
+    let input = args.get_value(0);
```

Comment:
Haven't checked it myself, but how do strings `"null"` and `"undefined"` behave here?